gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# PR #108, #109: Stack Matching Front End & Back End

### Description

We've implemented the front end and back end of `/eid/match`.

### Testing Directions

```
python manage.py makemigrations
python manage.py migrate
python manage.py pull_seatgeek_data
```

1.  Make 2-3 users and mark them as interested in/going to the same event.
1.  On `/`, click on that event link to navigate to `/eid/match`.
1.  "Thumbs up" or "Thumbs down" on users. See that there are no errors.

### Miscellaneous Comments

*   Sorry for the large PR.

### Checklist
